how-tos/maintain/push-your-new-version-to-production.md

```diff
@@ -0,0 +1,28 @@
+# Push your new version to production
+
+{% hint style="info" %}
+Forest Admin is using the [`.forestadmin-schema.json`](https://docs.forestadmin.com/developer-guide-agents-nodejs/under-the-hood/forestadmin-schema) file that is present beside your agent to reflect your model definition as well as the agent version.
```

Comment:
We should also here point out to problems that could occur if the schema is not up-to-date.

And also give pointers for people to check that the version of their schema is matching the version of their agent.